src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -1185,9 +1185,13 @@ public OnCompletion String(int value) {
             return null;
         }
 
+        public int getSerializedValue() {
```

Comment:
Why have a separate function here?

Reply:
Removed separate function.

Reply:
> @egieseke thanks! Could you add the unit test from #220 to `TestTransaction.java` as well?

Added unit test for Transaction and AppTransaction to TestTransaction.java.

Reply:
Removed this function and retested after removing JSONCreator annotation from string deserializer.